client/src/commons/Contexts/PersonalInfoStateContext.ts

```diff
@@ -1,32 +1,19 @@
 import {createContext} from 'react';
+import {personalInfoFormData} from 'models/Contexts/personalInfoContextData';
 
-import { initialPhoneNumberControl } from 'models/PhoneNumberControl';
-import {personalInfoContextData} from 'models/Contexts/personalInfoContextData';
-
-import { initialAddress } from './ClinicalDetailsContext';
-
-export const initialPersonalInfo: personalInfoContextData = {
-    phoneNumber: initialPhoneNumberControl,
-    additionalPhoneNumber: initialPhoneNumberControl,
-    contactPhoneNumber: initialPhoneNumberControl,
+export const initialPersonalInfo: personalInfoFormData = {
+    phoneNumber: '',
+    additionalPhoneNumber: '',
+    contactPhoneNumber: '',
     insuranceCompany: '',
-    address: initialAddress,
+    city: "",
```

Comment:
I don't usually comment on single quotes but all the other fields in this file have single quotes